pages/_app.js

```diff
@@ -37,6 +30,32 @@ export default function App({ Component, pageProps }) {
       : addRecipeToBookmarked(id);
   }
 
+  function handleAddRecipe(newRecipe) {
+    newRecipe.id = uid();
+    newRecipe.editable = true;
+    const updatedRecipes = [newRecipe, ...recipes];
+    setRecipes(updatedRecipes);
+  }
+
+  const [recipeToEdit, setRecipeToEdit] = useState(null);
+
+  function passRecipeToForm(recipe) {
+    setRecipeToEdit(recipe);
+  }
+
+  // ich möchte das recipe, wo der edit-befehl herkam, ändern.
+  // das ist ja im state zwischengespeichert.
+  // ich manipuliere hier aber den state (also das recipe wo es herkam) direkt ?? was man eigentlich nicht soll (funktioniert aber)
+  // und wenn ich das über den setter versuche, funktioniert es auch nicht
+  function handleEditRecipe(editedRecipe) {
+    recipeToEdit.title = editedRecipe.title;
+    recipeToEdit.ingredients = editedRecipe.ingredients;
+    recipeToEdit.preparation = editedRecipe.preparation;
+    recipeToEdit.usage = editedRecipe.usage;
+    recipeToEdit.symptoms = editedRecipe.symptoms;
+    setRecipeToEdit(null);
```

Comment:
Why do you set this setRecipeToEdit state to null? Maybe you can try to reset the form.